src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -268,7 +268,8 @@ public function enterHash(Cursor $cursor, $type, $class, $hasChild)
         } elseif (Cursor::HASH_RESOURCE === $type) {
             $prefix = $this->style('note', $class.' resource').($hasChild ? ' {' : ' ');
         } else {
-            $prefix = $class && !(self::DUMP_LIGHT_ARRAY & $this->flags) ? $this->style('note', 'array:'.$class).' [' : '[';
+            $unstyledPrefix = $class && !(self::DUMP_LIGHT_ARRAY & $this->flags) ? 'array:'.$class : '';
+            $prefix = $this->style('note', $unstyledPrefix).($unstyledPrefix ? ' [' : '[');
```

Comment:
I'm not sure I get the logic - why styling the empty string?

Reply:
Because if I don't trigger a coloration (even on an empty string), coloration is not enabled and the open bracket stay uncolored.

Reply:
the issue in https://github.com/symfony/symfony/issues/37674 is precisely complaining about that.